examples/crack_branching.cpp

```diff
@@ -23,25 +23,34 @@ int main( int argc, char* argv[] )
     MPI_Init( &argc, &argv );
 
     {
+        // ====================================================
+        //                  Setup Kokkos
+        // ====================================================
         Kokkos::ScopeGuard scope_guard( argc, argv );
 
         // FIXME: change backend at compile time for now.
```

Comment:
Let's remove this - I don't have any plans to support this for now

Reply:
I removed the comment "// FIXME: change backend at compile time for now."